src/pages/auth.js

```diff
@@ -58,6 +59,52 @@ export class Auth extends React.Component {
     return {status: 307, redirectTo: '/'};
   }
 
+  constructor() {
+    super();
+    this.state = {
+      registerClicked: false,
+      headerHidden: false
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.scrollHandler);
+  }
+
+  scrollHandler = () => {
+    if (this.state.registerClicked) return;
```

Comment:
Work with `scrollHandler()` is not done. So now this handler doesn't do anything. The idea is that if register form has been clicked once, handler shouldn't do any actions. But I probably should remove the handler from `window` instead of use this check for `this.state.registerClick`, shouldn't I?